### PR TITLE
feat(MeshAccessLog): support OpenTelemetry

### LINKED ISSUE #5998: OpenTelemetry support for accesslog
### Description

We should add otel backend support. See envoy [accesslog opentelemetry](https://github.com/envoyproxy/go-control-plane/tree/c077a74bfa9b647eb79489d4e423b3fc4f4ce47d/envoy/extensions/access_loggers/open_telemetry/v3) config.

https://github.com/kumahq/kuma/blob/286cd9c4e0077e5368812757a3d6a93f2af29b8b/pkg/plugins/policies/meshaccesslog/api/v1alpha1/meshaccesslog.go#L43-L46

https://github.com/kumahq/kuma/blob/286cd9c4e0077e5368812757a3d6a93f2af29b8b/pkg/plugins/policies/meshaccesslog/plugin/xds/configurer.go#L62-L71

### PR DESCRIPTION
**Pull request by michaelbeaumont**
### Checklist prior to review

- [x] Link to docs PR or issue --
- [x] Link to UI issue or PR --
- [x] Is the [issue worked on linked][1]? -- Closes #5998 
- [x] The PR does not hardcode values that might break projects that depend on kuma (e.g. "kumahq" as a image registry) --
- [x] The PR will work for both Linux and Windows, system specific functions like `syscall.Mkfifo` have equivalent implementation on the other OS --
- [x] Unit Tests --
- [x] E2E Tests -- none
- [x] Manual Universal Tests -- none
- [x] Manual Kubernetes Tests --
- [x] Do you need to update [`UPGRADE.md`](../blob/master/UPGRADE.md)? --
- [x] Does it need to be backported according to the [backporting policy](../blob/master/CONTRIBUTING.md#backporting)? --
- [x] Do you need to explicitly set a [`> Changelog:` entry here](../blob/master/CONTRIBUTING.md#submitting-a-patch) or add a `ci/` label to run fewer/more tests?

[1]: https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword

### CODE REVIEW

File: pkg/plugins/policies/meshaccesslog/api/v1alpha1/meshaccesslog.go [hunk 1]
```diff
@@ -54,6 +55,17 @@ type TCPBackend struct {
 	Address string `json:"address"`
 }
 
+// Defines an OpenTelemetry logging backend.
+type OtelBackend struct {
+	// Attributes can contain placeholders available on
+	// https://www.envoyproxy.io/docs/envoy/latest/configuration/observability/access_log
```
Comment (frzifus) [inline pkg/plugins/policies/meshaccesslog/api/v1alpha1/meshaccesslog.go:61]: It tells me: `Page not found :(` ^^

File: pkg/plugins/policies/meshaccesslog/api/v1alpha1/meshaccesslog.go [hunk 2]
```diff
@@ -54,6 +55,17 @@ type TCPBackend struct {
...
+	Attributes *[]JsonValue `json:"attributes,omitempty"`
```
Comment (frzifus) [inline pkg/plugins/policies/meshaccesslog/api/v1alpha1/meshaccesslog.go:62]: Is `*[]JsonValue` or did you mean `[]*JsonValue`

**Review by frzifus — APPROVED** [review]

File: pkg/plugins/policies/meshaccesslog/plugin/xds/clusters.go [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+func xdsEndpoint(endpoint LoggingEndpoint) core_xds.Endpoint {
+	return core_xds.Endpoint{
+		Target: endpoint.Address,
+		Port:   endpoint.Port,
+	}
+}
+
+func HandleClusters(backendEndpoints EndpointAccumulator, rs *core_xds.ResourceSet, proxy *core_xds.Proxy) error {
+	for backendEndpoint := range backendEndpoints.endpoints {
+		fmt.Println(backendEndpoint)
```
Comment (bartsmykla) [inline pkg/plugins/policies/meshaccesslog/plugin/xds/clusters.go:26]:
```suggestion
```

**Review by bartsmykla — APPROVED** [review]
one nit, but overall lgtm

**Pull request merged**